src/framework/theme/components/layout/layout.component.ts

```diff
@@ -442,8 +442,22 @@ export class NbLayoutComponent implements AfterViewInit, OnDestroy {
       .pipe(
         filter(() => this.withScrollValue),
       )
-      .subscribe((scrollable: boolean) => {
+      .subscribe((scrollable) => {
```

Comment:
Let's return type declaration here:
```suggestion
      .subscribe((scrollable: boolean) => {
```